database/migrations/2023_04_26_104633_create_landing_pages_tables_from_pages_table.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class CreateLandingPagesTablesFromPagesTable extends Migration
+{
+    public function up()
+    {
+        // Create landing_pages tables
+
+        DB::statement('CREATE TABLE landing_pages AS TABLE pages');
+        DB::statement('ALTER TABLE landing_pages
+                ADD COLUMN visit_nav_buy_tix_label TEXT,
+                ADD COLUMN visit_nav_buy_tix_link TEXT,
+                ADD COLUMN visit_hours_intro TEXT,
+                ADD COLUMN visit_members_intro TEXT,
+                ADD COLUMN visit_admission_intro TEXT');
```

Comment:
When you're issuing boilerplate SQL commands like these, I'd prefer you use Laravel's abstracted DB calls for table changes. Beacuse the CREATE TABLE command uses the AS TABLE syntax I think that line is ok to call as a DB::statement. Calls that add columns to existing tables is fairly straightforward and can instead be coded like this:

```
        Schema::table('landing_pages', function (Blueprint $table) {
            $table->text('visit_nav_buy_tix_label')->nullable();
            $table->text('visit_nav_buy_tix_link')->nullable();
            $table->text('visit_hours_intro')->nullable();
            $table->text('visit_members_intro')->nullable();
            $table->text('visit_admission_intro')->nullable();
        });
```

See the documentation here: https://laravel.com/docs/8.x/migrations#columns

Reply:
@web-dev-trev I had a go at refactoring the migrations. Take a look as a reference for future database work.